reference/forms/types/integer.rst

```diff
@@ -17,7 +17,6 @@ integers. By default, all non-integer values (e.g. 6.78) will round down
 | Rendered as | ``input`` ``number`` field                                            |
 +-------------+-----------------------------------------------------------------------+
 | Options     | - `grouping`_                                                         |
-|             | - `scale`_                                                            |
```

Comment:
IMHO we should better move it to the `Overridden` section, saying that the reverse transformation cast the value to an integer, ignoring the scale option at this point.

Reply:
Hi @HeahDude! Sorry for my late reply.
I'm not really convinced by the interest of keeping this point in the documentation if the user can not customize its behavior by any way. As any value that might have been set by the user to `scale` of `IntegerType` will be overridden by 0, it make sense for me to remove this section from its documentation.

Reply:
I understand, but this option is defined anyway and the type can still be extended to customize the transformation process. I would prefer to document the option and its overridden behavior instead of not mentioning anything.